pkg/sql/ttl/ttljob/ttljob_test.go

```diff
@@ -584,3 +584,177 @@ func TestRowLevelTTLJobRandomEntries(t *testing.T) {
 		})
 	}
 }
+
+// TestRowLevelTTLJobRandomEntries inserts random entries into a given table
+// and runs a TTL job on them.
+func TestRowLevelTTLJobExpirationExpressionRandomEntries(t *testing.T) {
```

Comment:
can we re-use the same test above to do this? otherwise, most of the logic seems duplicated and it'd be good to add a helper method to run these tests.

Reply:
Yeah, merged this test case into the test above.